README.md

```diff
@@ -1 +1,95 @@
-# React-bookstore
\ No newline at end of file
+# app-books
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+> This a personal project made using react and webpack
```

Comment:
- [ ] It would be nicer to add a descriptive description of the project to your README so that others learn more about your project.